src/components/TransactionPanel/BreakdownMaybe.js

```diff
@@ -16,13 +16,13 @@ const BreakdownMaybe = props => {
 
   return loaded ? (
     <div className={classes}>
-      <BookingBreakdown
+      <TeacherBookingBreakdown
```

Comment:
This is what confuse me. If we do not use BookingBreakdown component any more, why don't we just re-use and modify it instead of creating a new one.

Reply:
Oh sorry, I just replace that component to test, I'll add the condition to handle rendering the other component.